docs/examples/code-cells.ipynb

```diff
@@ -0,0 +1,960 @@
+    "\\end{equation}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Make sure to use at least version 0.1.6 of the `matplotlib-inline` package\n",
+    "(which is an automatic dependency of the `ipython` package).\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, the plots created with the \"inline\" backend have the wrong size.\n",
+    "More specifically, PNG plots (the default) will be slightly larger than SVG and PDF plots.\n",
+    "\n",
+    "This can be fixed easily by creating a file named `matplotlibrc`\n",
+    "(in the directory where your Jupyter notebooks live,\n",
+    "e.g. in this directory: [matplotlibrc](https://github.com/spatialaudio/nbsphinx/blob/9ee9da673b08e9db99ae1475fc83db6a62683e97/doc/matplotlibrc)\n",
+    "and adding the following line:\n",
+    "\n",
+    "    figure.dpi: 96\n",
+    "\n",
+    "If you are using Git to manage your files,\n",
+    "don't forget to commit this local configuration file to your repository.\n",
+    "Different directories can have different local configurations.\n",
+    "If a given configuration should apply to multiple directories,\n",
+    "symbolic links can be created in each directory.\n",
+    "\n",
+    "For more details, see\n",
+    "[Default Values for Matplotlib's \"inline\" Backend](https://nbviewer.org/github/mgeier/python-audio/blob/master/plotting/matplotlib-inline-defaults.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, plots are generated in the PNG format.\n",
+    "In most cases,\n",
+    "it looks better\n",
+    "if SVG plots are used for HTML output\n",
+    "and PDF plots are used for LaTeX/PDF.\n",
+    "This can be achieved by setting\n",
+    "[nbsphinx_execute_arguments](https://nbsphinx.readthedocs.io/en/0.9.7/configuration.html#nbsphinx_execute_arguments)\n",
+    "in your `conf.py` file like this:\n",
+    "\n",
+    "```python\n",
+    "nbsphinx_execute_arguments = [\n",
+    "    \"--InlineBackend.figure_formats={'svg', 'pdf'}\",\n",
+    "]\n",
+    "```\n",
+    "    \n",
+    "In the following example, `nbsphinx` should use an SVG image in the HTML output\n",
+    "and a PDF image for LaTeX/PDF output\n",
+    "(other Jupyter clients like JupyterLab will still show the default PNG format)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=[6, 3])\n",
+    "ax.plot([4, 9, 7, 20, 6, 33, 13, 23, 16, 62, 8]);"
```

Comment:
this renders as a `.png` in the built docs, identical to the `.png` in the following cell's output.